src/tribler-common/tribler_common/logger.yaml

```diff
@@ -64,12 +61,11 @@ handlers:
 # Root Logger Configuration
 root:
     level: NOTSET
-    handlers: [console, error_console, info_memory_handler, error_memory_handler]
-    propagate: yes
+    handlers: [stdout_handler, stderr_handler, info_memory_handler, error_memory_handler]
 
 # Module level configuration
-loggers:
-    TriblerGUI:
-        level: INFO
-        handlers: [console, error_console, info_memory_handler, error_memory_handler]
-        propagate: no
+# loggers:
+#    ExampleLoggerName:
+#        level: INFO
+#        handlers: [stdout_handler, stderr_handler, info_memory_handler, error_memory_handler]
+#        propagate: no
```

Comment:
**Nit:** Commented part can be safely removed.

Reply:
It indeed can be removed, but I left it as an example of loggers section structure. It is primarily useful for suppressing some overly verbose logger. For example, a developer can set DEBUG verbosity in logs, but want to turn off `TunnelDispatcher` and `PyQt5.uic` loggers to reduce spam in console. Probably it is better for me to extend the comment and add detailed example for this use-case.